### PR TITLE
[Wear] Enable play/pause on the Player

### PR DESCRIPTION
**Pull request by kul3r4**
Adds logic for enabling the button play/pause.

### CODE REVIEW

File: Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt [hunk 1]
```diff
@@ -80,10 +83,20 @@ private fun PlayerScreen(
     PlayerScreen(
         mediaDisplay = {
             if (playerUiState != null) {
-                playerUiState.episodePlayerState.currentEpisode?.let {
+                if (playerUiState.episodePlayerState.currentEpisode != null) {
```
Comment (arriolac) [inline Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt:86]: 2 null checks are being done here. You can simplify this to:

```
val episode = playerUiState.episodePlayerState.currentEpisode
if (episode != null) {
     TextMediaDisplay(
         title = episode.podcastName,
         subtitle = episode.title
     )
} else {
    TextMediaDisplay(
          title = it.podcastName,
          subtitle = it.title
          title = "Nothing to play",
          subtitle = ""
      )
}
```

File: Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt [hunk 2]
```diff
@@ -80,10 +83,20 @@ private fun PlayerScreen(
...
+                                title = episode.podcastName,
+                                subtitle = episode.title
+                            )
+                        }
+                    }
+                } else {
                     TextMediaDisplay(
-                        title = it.podcastName,
-                        subtitle = it.title
+                        title = "Nothing to play",
```
Comment (arriolac) [inline Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt:98]: This string should be added to `strings.xml` and read from `stringResource`

File: Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt (hunk 1 above)
Comment (kul3r4) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt:86]: In my check I am also checking that the tile is not "" as I want to show "Nothing to play" instead (data class PlayerEpisode sets the title to "")

Comment (arriolac) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt:86]: How about?

```
val episode = playerUiState.episodePlayerState.currentEpisode
if (episode != null && episode.title.isNotEmpty()) {
     TextMediaDisplay(
         title = episode.podcastName,
         subtitle = episode.title
     )
} else {
    TextMediaDisplay(
          title = it.podcastName,
          subtitle = it.title
          title = "Nothing to play",
          subtitle = ""
      )
}
```

File: Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt [hunk 3]
```diff
@@ -72,6 +74,7 @@ fun PlayerScreen(
 @Composable
 private fun PlayerScreen(
     playerUiState: PlayerUiState?,
```
Comment (yschimke) [inline Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt:76]: PlayerUiState can be non nullable.

We might want to change the model to reflect no loaded item, rather then "" being no item.

Comment (kul3r4) [reply Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt]: thanks, removed the ?

**Pull request merged**